CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.13.2 (March 20, 2024)
+
+ENHANCEMENTS:
+
+* `resources/pipes_workspace`: Add support for passing `desired_state` attribute to update workspace request. Add attribute `state_reason`.
+* `resources/pipes_workspace_pipeline`: Add support for passing `desired_state` attribute to update pipeline request. Add attributes `state` and `state_reason`.
+
```

Comment:
@sidr0cker you should also mention the fixes in the docs for datatanks.

Reply:
@MichaelBurgess we generally do not mention doc updates in the changelog. do we want to do that going forward?